sql/core/src/main/scala/org/apache/spark/sql/execution/command/ddl.scala

```diff
@@ -53,12 +54,19 @@ case class CreateDatabase(
   extends NativeDDLCommand(sql) with Logging
 
 case class CreateFunction(
+    databaseName: Option[String],
     functionName: String,
     alias: String,
     resources: Seq[(String, String)],
     isTemp: Boolean)(sql: String)
   extends NativeDDLCommand(sql) with Logging
 
+case class DropFunction(
+    functionName: String,
```

Comment:
Because from Hive DDL [manual](https://cwiki.apache.org/confluence/display/Hive/LanguageManual+DDL#LanguageManualDDL-Create/Drop/ReloadFunction), it doesn't explicitly mention database name parameter for drop function.

But from the parser rule, it uses the same parsing rule as create function DDL command. So it should support database name. Let me add it.